src/index.html

```diff
@@ -13,6 +13,38 @@
     />
```

Comment:
The `href` attribute should point to a `.css` file, not a `.scss` file. SCSS files are preprocessed into CSS files which browsers can understand. Make sure to compile your SCSS and reference the resulting CSS file here.